autogen/agentchat/contrib/critic_agent.py

```diff
@@ -0,0 +1,55 @@
+from typing import Callable, Dict, Literal, Optional, Union
+
+from autogen.agentchat.conversable_agent import ConversableAgent
+from autogen.runtime_logging import log_new_agent, logging_enabled
+
+
+class CriticAgent(ConversableAgent):
+    """
+    An agent for creating list of criteria for evaluating the utility of a given task.
+    """
+
+    DEFAULT_SYSTEM_MESSAGE = """You are a helpful assistant. You suggest criteria for evaluating different tasks. They should be distinguishable, quantifiable and not redundant.
+    Convert the evaluation criteria into a dictionary where the keys are the criteria.
+    The value of each key is a dictionary as follows {"description": criteria description , "accepted_values": possible accepted inputs for this key}
+    Make sure the keys are criteria for assessing the given task.  "accepted_values" include the acceptable inputs for each key that are fine-grained and preferably multi-graded levels. "description" includes the criterion description.
+    Return the dictionary."""
+
+    DEFAULT_DESCRIPTION = "An AI agent for creating list criteria for evaluating the utility of a given task."
+
+    def __init__(
+        self,
+        name="critic",
+        system_message: Optional[str] = DEFAULT_SYSTEM_MESSAGE,
+        llm_config: Optional[Union[Dict, bool]] = None,
+        max_consecutive_auto_reply: Optional[int] = None,
+        human_input_mode: Optional[str] = "NEVER",
+        description: Optional[str] = DEFAULT_DESCRIPTION,
```

Comment:
For the arguments that you don't need to alter the default, don't list them explicitly. That will make it easier to maintain.
Also, the current way of writing the constructor will remove a few kwargs and stop them from being passed to super(). Is that intentional?